services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationForm.tsx

```diff
@@ -162,209 +161,201 @@ const MajorMineApplicationForm: React.FC<MajorMineApplicationFormProps> = ({
 
   return (
     <div>
-      <FormWrapper
-        name={FORM.ADD_MINE_MAJOR_APPLICATION}
-        initialValues={{
-          primary_documents: [],
-          spatial_documents: [],
-          supporting_documents: [],
+      <Row>
```

Comment:
it thinks this is a large diff but I swear all I did was take out FormWrapper and the handleSubmit prop.